app/components/task-tabs.hbs

```diff
@@ -3,7 +3,7 @@
     class='task-tabs__dropdown-container
       {{if @showTasks "task-tabs__dropdown-active"}}'
   >
-    {{#each @taskStatusList as |status|}}
+    {{#each this.removedTaskTabs as |status|}}
```

Comment:
Can we use any other name instead `removedTaskTabs`

Reply:
done